app/helpers/attachments_helper.rb

```diff
@@ -1,4 +1,10 @@
 module AttachmentsHelper
+  include Rails.application.routes.url_helpers
```

Comment:
I think these should go in `GovspeakRenderer`, because this module will normally have access to these when called within a normal request/response context.